be1-go/hub/standard_hub/message_handling.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const publishError = "failed to publish: %v"
 
-// handleRootChannelPublishMesssage handles an incominxg publish message on the root channel.
+// handleRootChannelPublishMesssage handles an incoming publish message on the root channel.
 func (h *Hub) handleRootChannelPublishMesssage(sock socket.Socket, publish method.Publish) error {
```

Comment:
It's not from you of course but it could be occasion to correct the typo in function name and comment